lib/jekyll-mentions.rb

```diff
@@ -18,7 +18,7 @@ def generate(site)
 
     def mentionify(page)
       return unless page.content.include?('@')
-      page.content = @filter.mention_link_filter(page.content)
+      page.content = @filter.mention_link_filter(page.content, '/', nil, /[a-zA-Z0-9][a-zA-Z0-9-]*/)
```

Comment:
Would it make more sense to use `/#{HTML::Pipeline::UsernamePattern}/i` for future compatibility? At the very least, we should probably use the library's default regex with an `i` flag, rather than writing our own, I'd think?